content/en/agent/faq/ec2_imdsv2_transition_payload_enabled.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Enablement IMDSv2 by default
+aliases:
+  - /ec2_imdsv2_transition_payload_enabled
+---
+
+# IMDSv2 Default Behavior & Host Metadata Enhancement
+
+From 7.64.0, the Agent now enables IMDSv2 by default for enhanced security and improved metadata handling. In addition, a new field, `legacy-resolution-hostname`, has been added to the host metadata payload. This field is used to capture the AWS instance hostname using legacy resolution logic (i.e. when IMDSv2 was disabled) and is only included when needed.
+
+> **Important:** The hostname displayed by the Agent **may** change if the Agent previously did not show the instance ID as the hostname, however it will continue to use the original hostname to tag the different metrics sent by the Agent.
```

Comment:
> The hostname displayed by the Agent

It might be good to be more specific on what we mean by Agent here.

Alternatively, maybe we can state something like this:
> To ensure backwards compatibility, the hostname displayed in Datadog may not be an instance ID if the host previously existed with a different hostname before the agent upgrade